radix-engine/src/track/interface.rs

```diff
@@ -170,12 +170,22 @@ pub trait SubstateStore {
 #[derive(Debug, Clone, Copy)]
 pub enum StoreAccess {
     /// Some substate was read from database.
-    ReadFromDb(usize),
+    ReadFromDb(NodeId, usize),
     /// Non-existent substate was read from database.
-    ReadFromDbNotFound,
+    ReadFromDbNotFound(NodeId),
```

Comment:
If we're adding `NodeId`, is there a reason not to include the full substate path? `NodeId::PartitionNum::SubstateKey`

Reply:
It's primarily used for costing breakdown and so far we only need node id. 

Can definitely add more in future.